docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+  neo4j:
+    environment:
+      # The initial neo4j username MUST BE "neo4j" if a db user has not been previously created.
+      # The initial neo4j passworrd is tied to the docker volume
+      # If you want to update the password you must update the account in neo4j
+      # Then, you need to relaunch cartography process so it has the updated password
+      - NEO4J_AUTH=${NEO4J_USERNAME}/${NEO4J_PASSWORD}
+    image: neo4j:3.5
+    healthcheck:
+      test: curl -i http://127.0.0.1:7474 2>&1 | grep -c -e '200 OK'
+    ports:
+      - "7687:7687" # host:container
+      - "7474:7474"
+    volumes:
+      - $HOME/neo4j/data:/data
```

Comment:
This assumes users will want to store this test data in their own directory.
What do you think of using a volume instead?

Reply:
Every time we spin up the docker container, cartography would have to resync against the aws account. I think this is pretty expensive to do.

I can make this another docker-compose variable possibly like `${NEO4J_VOLUME}` Thoughts?